stemcell_builder/stages/image_install_grub/apply.sh

```diff
@@ -156,5 +170,12 @@ else
   exit 2
 fi
 
+md5_password=`run_in_chroot ${image_mount_point} "if which grub-md5-crypt > /dev/null;then echo -e '${random_password}\n${random_password}' | grub-md5-crypt | grep -e '^\\$1\\$.*'; fi"`
```

Comment:
@xingzhou and also here:

1) why we need to check if grub-md5-crypt is present, can we just call it?
2) lets put password line when we writing template above (line  150,162)
3) always chmod/chown grub.conf